go/batch-submitter/bindings/ctc/canonical_transaction_chain.go

```diff
@@ -0,0 +1,1522 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Should this be stored in the index, or ignored and generated during the build process? 
I'm not sure of the correct convention here.

Reply:
I have previously been just committing in the bindings as to not need to pull things from the contracts package into the docker image. I think either way is fine

Reply:
@maurelian yes i think ideally these would be generated as part of the build process, the main barrier to doing so deterministically would require pinning the version of abigen across all dev/ci machines (probably via a docker compilation image). for now i'd be in favor of just committing to keep it simple, but i can look into doing this in one fell swoop for all of our go services

Reply:
FWIW, I have no strong opinion. I'm a tourist in go-land here.